{
    "title": "The django.util.http.is_safe_url function in Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly validate URLs, which allows remote attackers to conduct open redirect attacks via a malformed URL, as demonstrated by http:djangoproject.com.",
    "score": 4.3,
    "severity": "MEDIUM",
    "cwe_id": "CWE-20",
    "cvss_vector": "CVSS:2.0/AV:N/AC:M/Au:N/C:N/I:P/A:N",
    "cpe_match": [
        "cpe:2.3:o:canonical:ubuntu_linux:13.10:*:*:*:*:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:12.10:*:*:*:*:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:14.04:*:*:*:lts:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:12.04:-:lts:*:*:*:*:*",
        "cpe:2.3:o:canonical:ubuntu_linux:10.04:-:lts:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.12:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.9:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.10:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.6:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.4:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.5:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.2:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.11:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.7:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.1:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4.8:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.4:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.7:beta1:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.7:beta3:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.7:beta2:*:*:*:*:*:*",
        "cpe:2.3:o:opensuse:opensuse:12.3:*:*:*:*:*:*:*",
        "cpe:2.3:o:opensuse:opensuse:13.1:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6:beta4:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6:beta2:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6.3:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6.4:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6:-:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6.1:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6.2:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6:beta1:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.6:beta3:*:*:*:*:*:*",
        "cpe:2.3:o:debian:debian_linux:8.0:*:*:*:*:*:*:*",
        "cpe:2.3:o:debian:debian_linux:7.0:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5.7:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5.1:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5.3:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5.4:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5:beta:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5.5:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5.2:*:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5:alpha:*:*:*:*:*:*",
        "cpe:2.3:a:djangoproject:django:1.5.6:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "technique_id": "T1033",
            "confidence": "2.2",
            "technique_name": "System Owner/User Discovery"
        },
        {
            "technique_id": "T1048",
            "confidence": "2.3",
            "technique_name": "Exfiltration Over Alternative Protocol"
        },
        {
            "technique_id": "T1112",
            "confidence": "2.3",
            "technique_name": "Modify Registry"
        },
        {
            "technique_id": "T1113",
            "confidence": "2.1",
            "technique_name": "Screen Capture"
        },
        {
            "technique_id": "T1204",
            "confidence": "2.1",
            "technique_name": "User Execution"
        },
        {
            "technique_id": "T1486",
            "confidence": "2.3",
            "technique_name": "Data Encrypted for Impact"
        }
    ]
}
